tutorials/btp-kyma-onb-02-func/btp-kyma-onb-02-func.md

```diff
@@ -1,305 +1,199 @@
+    - **Name**: `orders-service`
+    - **app**: `orders-deployment`
+    - Add a port using the following parameters:
+        - **Name**: `orders-port`
+        - **Protocol**: `TCP`
+        - **Port**: `80`
+        - **Target Port**: `8080` (or other)
+        - **Application Protocol**: `http`
 
+You created a new service, called **orders-service**.
 
 ### Expose the microservice
 
-You cannot access and test your new orders-service yet from outside the cluster. 
-
-To expose the microservice, you must create an **API Rule** for it, just like when you exposed our Function.
-
-1. In your Kyma dashboard, go to **Discovery and Network --> API Rules**.
-
-2. Click on `+ Create API Rule`. 
-
-3. A popup window opens. Keep the preset **Simple**.
-
-4. Provide the following values in the Create API Rule view:
-
-    - Name: orders-apirule
-
-    - HTTP Request Timeout: 30 (or any up to 3900)
-   
-    - Service Name: orders-service (from dropdown list box).
-
-    - Port: 80
+You cannot access and test your new `orders-service` yet from outside of the cluster. To expose the microservice, first, you must create an **API Rule** for it, just like you did to expose your Function.
 
-    In the section "Gateway", keep the pre-configured Namespace and Name. 
+1. Go to **Discovery and Network > API Rules** and choose **Create**.
 
-    - Choose your host from the Host dropdown. 
-  
-    - Replace the wildcard (*) with a name, for example: orders-host
-
-    Keep the values in the section "Rules".
-
-    ![api rule create](images/5_5_apirule_create.png)
-
-5. Choose Create.
-
-    A new API Rule will be created. Status must be **OK**.
-
-6. You can open the link by clicking on the provided URL. The provided link will be empty because the underlying docker image has no home page. 
-
-    The URL has to be extended with the String **/orders**.
-
-    For example: 
+2. Fill in the form in the **Create API Rule** view using the following details and choose **Create**
 
-    URL: `https://orders-host.c-123456.kyma.ondemand.com/` has to become 
-    
-    URL: `https://orders-host.c-123456.kyma.ondemand.com/orders` 
+   - **Name**: `orders-apirule`
+   - **HTTP Request Timeout**: `30` (or any up to `3900`)
+   - **Service Name**: `orders-service`
+   - **Port**: `80`
+   - Leave the pre-defined details in the **Gateway** section
+   - **Host**: choose your host from the dropdown menu and replace the wildcard (*) with a name, for example, `orders-host`
+   - Leave the pre-defined details in the **Rules** section
+   - **Methods**: choose `GET` and `POST`
 
-    ![api rule start](images/5_6_apirule_start.png)
+3. The `orders-apirule` is created. Wait for the **Status** to be `OK`.
 
+4. Click on the **Host** URL to execute the Service in a browser window. A browser will open but the link will be empty because the underlying Docker image has no home page. Extend the URL by adding the **/orders** string.
```

Comment:
```suggestion
4. Click on the **Host** URL to execute the Service in a browser window. A browser will open, but the link will be empty because the underlying Docker image has no homepage. Extend the URL by adding the **/orders** string.
```